tokens/package.json

```diff
@@ -15,8 +15,10 @@
   },
   "main": "dist/index.css",
   "devDependencies": {
-    "@adobe/spectrum-tokens": "12.23.3",
+    "@adobe/spectrum-tokens": "13.0.0-beta.8",
     "@nxkit/style-dictionary": "^4.0.0",
+    "concat-cli": "^4.0.0",
+    "npm-run-all": "^4.1.5",
```

Comment:
I think these might be added from an old version. We don't use npm-run-all or concat-cli anymore for token building.